ArduCopter/APM_Config.h

```diff
@@ -57,3 +57,4 @@
 //#define USERHOOK_MEDIUMLOOP userhook_MediumLoop();        // for code to be run at 10hz
 //#define USERHOOK_SLOWLOOP userhook_SlowLoop();            // for code to be run at 3.3hz
 //#define USERHOOK_SUPERSLOWLOOP userhook_SuperSlowLoop();  // for code to be run at 1hz
+#define DEVO_TELEM_ENABLED ENABLED
```

Comment:
I think this one should follow the pattern of the others:
* Use a ifndef inside the `config.h` file
* Use a `//#define DEVO_TELEM_ENABLED DISABLED` inside `ArduCopter/APM_Config.h`

Reply:
if it will be disabled by default, then it will be useless for community. #ifdef wil be added

Reply:
on `config.h` do something like:
```
//use this to completely disable FRSKY TELEM
#ifndef FRSKY_TELEM_ENABLED
  #  define FRSKY_TELEM_ENABLED          ENABLED
#endif
```

And then on `APM_Config.h`
`//#define DEVO_TELEM_ENABLED DISABLED` just after the line `//#define FRSKY_TELEM_ENABLED DISABLED`

Reply:
This way it is enable by default and it can be easily be disabled by uncommenting the line in `APM_Config.h` just like all the other options